### PR TITLE
Update RELEASE_MANAGEMENT.md

### PR DESCRIPTION
**Pull request by Dimitar5555**
Successor to #5029

### CODE REVIEW

**Review by Jean-BaptisteC — APPROVED** [review]

**Comment by biodranik** [comment]
Please next time do not create new pull requests. Learn how git rebase and amending git commits work, and update/edit your existing PR.

File: docs/RELEASE_MANAGEMENT.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 ### Upload metadata to AppStore
```
Comment (biodranik) [inline docs/RELEASE_MANAGEMENT.md:5]: Let's merge these two sections into one: Upload metadata and screenshots to AppStore.

**Comment by Dimitar5555** [comment]
> Please next time do not create new pull requests. Learn how git rebase and amending git commits work, and update/edit your existing PR.

Sorry about that. I tried to rebase it using Github Desktop but it stopped responding while downloading the repository twice in a row..

**Comment by biodranik** [comment]
It's better to learn rebase from the command line. UI utilities may not work properly.

**Pull request merged**